### ISSUE TITLE
Error when loading help pages in browser on Windows

### ISSUE THREAD

**Issue by alex-konovalov** [gap-system/gap#361]
After `SetHelpViewer("browser")`, GAP help does not work any more: it fails to open help page with the error message:

"Opening help page in default windows browser ... /usr/bin/cygstart.exe: error while loading shared libraries: ?: cannot open shared object file: No such file or directory"

Workaround for the users experiencing this problem: switch back to default `SetHelpViewer("screen");`

**Comment by ChrisJefferson** [comment]
I tried building `Browse` on cygwin, gap 4.7.7 and master, and both failed to compile. Is there some secret magic I have to do? (or maybe there are updates need for windows 10 / 64-bit cygwin?)

**Comment by alex-konovalov** [comment]
Are you using Win 10 and 64-bit Cygwin? In this case I cant help yet. But the problem is reproducible when Browse is not loaded too.

**Comment by ChrisJefferson** [comment]
I have fixed win 10 / 64-bit cygwin. After doing this, the code works fine if I build GAP and in my `cygfix` branch. Where abouts is it broken?

**Comment by alex-konovalov** [comment]
This may be relevant: https://cygwin.com/ml/cygwin/2012-07/msg00264.html 

I'm afraid that the situation just became worse since that time...

**Comment by alex-konovalov** [comment]
I've found a way to a workaround - the following opens the html page in the default browser:
```
gap> Exec("start c:\\gap4r9\\doc\\ref\\chap65.html");
```
Sadly, one can not navigate to the exact location since this fails
```
gap> Exec("start c:\\gap4r9\\doc\\ref\\chap65.html#X82C63644805EB1EE");
```
but the same limitation on Windows was there before.

**Comment by fingolfin** [comment]
Fixed by @alex-konovalov in PR #603
